rest-service/manager_rest/rest/resources_v3_1/manager.py

```diff
@@ -229,6 +229,60 @@ class RabbitMQBrokers(SecuredResource):
     @paginate
     @authorize('broker_get')
     def get(self, pagination=None):
+        """List brokers from the database."""
         return get_storage_manager().list(
             models.RabbitMQBroker,
         )
+
+    @exceptions_handled
+    @authorize('broker_add')
+    @marshal_with(models.RabbitMQBroker)
+    def post(self):
+        """Add a broker to the database."""
+        _broker = rest_utils.get_json_and_verify_params({
+            'name': {'type': unicode},
+            'address': {'type': unicode},
```

Comment:
Add validation to ensure `address` is in `networks`?

Reply:
in the future i think we should get rid of `address` entirely because (even currently) it does nothing, doesn't it...

Reply:
Note: we'll be validating this.